cli/src/cli_util.rs

```diff
@@ -1274,7 +1274,7 @@ See https://github.com/martinvonz/jj/blob/main/docs/working-copy.md#stale-workin
             .retain(|change_id, _commits| !removed_conflicts_by_change_id.contains_key(change_id));
 
         // TODO: Also report new divergence and maybe resolved divergence
-        let mut fmt = ui.stderr_formatter();
+        let mut fmt = ui.status();
```

Comment:
Perhaps, `ui.status()` can just return `LabeledWriter`? I think callers who depend on the formatter API would probably want to do early return if `ui.status()` is suppressed.

Reply:
Interesting. I had not even considered that some of the call sites would want to return early if `--quiet` was passed. What's the advantage to returning `LabeledWriter`? I think one difference is that we can add a `status` label to it, but I'm not sure how useful that is in practice. It we want to use it consistently, we'd have to remember to add that label in the places where we would return early if `--quiet` was passed.

Reply:
I just thought the user might want to change the color of the status message, but yeah, I agree it would be tedious if we had to push/pop the label at call sites.

Perhaps, early return can be signaled if `.status_formatter()` returned `Option<Formatter>`, and `.status() -> io::Write`?